js/interface.js

```diff
@@ -838,6 +838,16 @@ $('[data-push-save]').on('click', function() {
   savePushData();
 });
 
+// Scroll accordion tab to the top
+$(document).ready(function (){
```

Comment:
same question as on the other ticket, why is `$(document).ready` required here?

Reply:
It is not necessary but I used it to make sure that the document is fully loaded.
If it is not need, I will change it without $(document).ready.

Reply:
If the `.panel-collapse` element is printed in the interface.html then you won't need to wait for the document ready event. 

Furthermore, if you need to wait for such event please use `Fliplet().then(function () {})`

Reply:
fixed